#!/usr/bin/env python
PACKAGE = "ainstein_radar_drivers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add( "min_range", double_t, 0, "Minimum target range, in meters", 0.0, 0.0, 80.0 )
gen.add( "max_range", double_t, 0, "Maximum target range, in meters", 100.0, 0.0, 80.0 )

gen.add( "min_angle", double_t, 0, "Minimum azimuth angle, in degrees", 60.0, 0.0, 60.0 )
gen.add( "max_angle", double_t, 0, "Maximum azimuth angle, in degrees", 60.0, 0.0, 60.0 )

exit(gen.generate(PACKAGE, "ainstein_radar_drivers", "ZoneOfInterestT79"))
